scripts/globus/globus_cleanup.py

```diff
@@ -0,0 +1,207 @@
+import globus_sdk
+from globus_sdk import AuthClient, GroupsClient
+from globus_sdk.scopes import GroupsScopes
+import subprocess
+import sys
+import os
+import utils
+
+# Define your client ID and client secret
+CLIENT_ID = "f8d0afca-7ac4-4a3c-ac05-f94f5d9afce8"  # NATIVE
```

Comment:
Should this be hardcoded?

Reply:
Yes, it should be. Good catch though, the native client id still requires users to authenticate with their own credentials, it will make a request on behalf of the user. This should be associated with DataFed.

Reply:
I will update the file with a comment.